doc/how-to-generate.md

```diff
@@ -56,7 +56,8 @@ As autorest.az support auto detect full/incremental generation mode. The rules a
  ```
 
 ## Full codegen to cli
-Coming soon
+You can use `--target-mode=core` change the `--azure-cli-extension-folder` to `--azure-cli-folder` to generate cli modules. see [here](how-to-generate-with-different-options.md#generate-cli-main-repo-modules-with-default-option) for more details.  
+You also use other interesting options when generating the cli. see [here](how-to-generate-with-different-options.md#what-kind-of-options-do-we-have) for more details.  
 
```

Comment:
Is it possible that we can orginize the content more clearly and readable? Currently it seems more divisive.. Reader need to find info in two parts but the link is only put in core session, although the linked page also contains information about extension. 

Here is my suggestion, brief the scenarios we supported in current doc in first part, and provide the link to the page, which including option details and useful conbintation about options. 
In current doc, we still list the scenarios about core/ext, full/inc with their sample.

Reply:
I don't intro other options at all here. the change from --azure-cli-extension-folder to --azure-cli-folder is necessary for target-mode=core